consensus/src/config.rs

```diff
@@ -31,3 +31,5 @@ pub const SECOND_REDUCTION_COMMITTEE_SIZE: usize = 64;
 pub const CONSENSUS_DELAY_MS: u64 = 1000;
 
 pub const DEFAULT_BLOCK_GAS_LIMIT: u64 = 5 * 1_000_000_000;
+
+pub const RELAX_ITERATION_THRESHOLD: u8 = 10;
```

Comment:
I strongly believe this value should be much higher.
Maybe we can discuss this in the Rolling Finality PR

Reply:
Let's adjust this config with a new PR once we gain clarity. Have in mind that a higher value may result in the verification of more certificates during the block acceptance, and we've observed that this has an impact on performance.

 See also https://github.com/dusk-network/rusk/issues/1149#issuecomment-1829391702